src/plugins/wb-randomize/wb-randomize.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @title WET-BOEW Randomize
+ * @overview This plugin randomly picks one of the child component to be shown on the browser.
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author @masterbee
+ */
+
+( function( $, wb ) {
+"use strict";
+
+$( ".provisional[data-wb-randomize]" ).each( function() {
+	var $elm = $( this ),
+		command = $elm.data( "wb-randomize" ),
+		shuffle = command.shuffle,
+		selector = command.selector,
+		number = command.number,
+		context = $elm,
+		selected = $( selector, context );
+
+	selected.shuffle = wb.shuffle;
+
+	if ( shuffle ) {
+		selected = selected.shuffle();
+	}
+
+	selected.rand = wb.rand;
```

Comment:
I don't think we need that association

Reply:
without this, random function would not work.